compose/.apps/transmissionvpn/transmissionvpn.labels.yml

```
@@ -9,3 +9,6 @@ services:
       com.dockstarter.appvars.transmissionvpn_port_51413: "51413"
       com.dockstarter.appvars.transmissionvpn_port_6881: "6881"
       com.dockstarter.appvars.transmissionvpn_port_9091: "9091"
+      com.dockstarter.appvars.transmissionvpn_download_dir: "/downloads/completed"
+      com.dockstarter.appvars.transmissionvpn_incomplete_dir: "/downloads/incomplete"
+      com.dockstarter.appvars.transmissionvpn_watch_dir: "/downloads/watch"
```

```suggestion
      com.dockstarter.appvars.transmissionvpn_download_dir: "/storage/downloads/completed"
      com.dockstarter.appvars.transmissionvpn_incomplete_dir: "/storage/downloads/incomplete"
      com.dockstarter.appvars.transmissionvpn_watch_dir: "/storage/downloads/watch"
```
Also these need to be sorted alphabetically in with the rest of the labels (ex: `transmissionvpn_download_dir` should be up above ports and such)